charts/postgres-operator/values-crd.yaml

```diff
@@ -53,9 +53,11 @@ configKubernetes:
   cluster_domain: cluster.local
   # additional labels assigned to the cluster objects
   cluster_labels:
-      application: spilo
+    application: spilo
   # label assigned to Kubernetes objects created by the operator
   cluster_name_label: cluster-name
+  # additional annotations to add to every database pod
+  custom_pod_annotations:
```

Comment:
```suggestion
  # custom_pod_annotations: 
  #   keyb: valueb
  #   keyz: valuez
```

Reply:
I forgot to mention that there is also an example [manifest](https://github.com/zalando/postgres-operator/blob/master/manifests/postgresql-operator-default-configuration.yaml) for an `OperatorConfiguration` resource. Could you add the field there, too? Thanks. Same layout as I suggested here.